doc/source/data/examples/batch_inference_object_detection.ipynb

```diff
@@ -0,0 +1,361 @@
+   "source": [
+    "Then, we use the `map_batches` API to apply the model to the whole image data set. Here, the `compute` argument indicates the number of concurrent Ray actors that run the model, and the `batch_size` argument indicates the number of images in each batch."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ds = ds.map_batches(\n",
+    "    ObjecytDetectionModel,\n",
+    "    compute=ray.data.ActorPoolStrategy(size=4),\n",
+    "    batch_size=4,\n",
+    ")"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Lastly, we can use `ds.show` to inspect samples in the result, or the the `ds.write_xxx` APIS to write results to external storages."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "2023-05-09 14:23:31,331\tINFO streaming_executor.py:91 -- Executing DAG InputDataBuffer[Input] -> ActorPoolMapOperator[ReadImage->MapBatches(ObjecytDetectionModel)]\n",
+      "2023-05-09 14:23:31,332\tINFO streaming_executor.py:92 -- Execution config: ExecutionOptions(resource_limits=ExecutionResources(cpu=None, gpu=None, object_store_memory=None), locality_with_output=False, preserve_order=False, actor_locality_enabled=True, verbose_progress=False)\n",
+      "2023-05-09 14:23:31,332\tINFO streaming_executor.py:94 -- Tip: For detailed progress reporting, run `ray.data.DataContext.get_current().execution_options.verbose_progress = True`\n",
+      "2023-05-09 14:23:31,346\tINFO actor_pool_map_operator.py:114 -- ReadImage->MapBatches(ObjecytDetectionModel): Waiting for 4 pool actors to start...\n"
+     ]
+    },
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "38b198e766394b6ca0e2101cc0393e7e",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Running 0:   0%|          | 0/200 [00:00<?, ?it/s]"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "2023-05-09 14:27:58,835\tINFO streaming_executor.py:149 -- Shutting down <StreamingExecutor(Thread-6787, started daemon 13735325696)>.\n",
+      "2023-05-09 14:27:58,997\tWARNING actor_pool_map_operator.py:264 -- To ensure full parallelization across an actor pool of size 4, the specified batch size should be at most 0. Your configured batch size for this operator was 16.\n"
+     ]
+    },
+    {
+     "ename": "KeyboardInterrupt",
+     "evalue": "",
+     "output_type": "error",
+     "traceback": [
+      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
+      "\u001b[0;31mKeyboardInterrupt\u001b[0m                         Traceback (most recent call last)",
```

Comment:
There's an error in this cell. I think you probably want to go through and audit the outputs.

Reply:
I accidentally committed this. Do we usually also commit the outputs of each cell? Or should I just commit a clean notebook?

Reply:
Checked other examples. We do commit outputs.